ext/civiimport/Civi/BAO/Import.php

```diff
@@ -251,4 +251,32 @@ private static function getAllFields(string $tableName): array {
     return array_merge(self::getFieldsForTable($tableName), self::getSupportedFields());
   }
 
+  /**
+   * Defines the default key as 'id'.
+   *
+   * @return array
+   */
+  public function keys() {
+    static $keys;
+    if (!isset($keys)) {
+      $keys = ['_id'];
+    }
+    return $keys;
+  }
+
+  /**
+   * Tells DB_DataObject which keys use autoincrement.
+   * 'id' is autoincrementing by default.
+   *
+   *
+   * @return array
+   */
+  public function sequenceKey() {
+    static $sequenceKeys;
+    if (!isset($sequenceKeys)) {
+      $sequenceKeys = [$this->getFirstPrimaryKey(), TRUE];
```

Comment:
I feel like we could also just return `['_id', TRUE]` here - seems overly complicated to be setting statics & calling functions when the known thing is that it is `_id`

Reply:
ok updated @eileenmcnaughton